manifests/0000_12_etcd-operator_00_staticsync-rolebinding.yaml

```diff
@@ -0,0 +1,26 @@
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  namespace: openshift-etcd
+  name: cluster-etcd-operator-staticsync
+rules:
```

Comment:
naming here needs to be revisited. this role is used more generically by static pods to communicate with scaling-lock